.ci/azure-pipelines-package.yml

```
@@ -193,20 +193,34 @@ jobs:
   pool:
     vmImage: 'ubuntu-latest'
 
+  variables:
+  - name: JellyfinVersion
+    value: 0.0.0
```

You can set the value to a replace expression

```suggestion
    value: $[replace(variables['Build.SourceBranch'],'refs/tags/v','')]
```

just like what was done here: https://github.com/jellyfin/jellyfin-apiclient-java/blob/master/.ci/azure-pipelines.yml#L65